LetSwift/ApplicationViewControllers/Onboarding/OnboardingViewController.swift

```diff
@@ -66,6 +68,15 @@ final class OnboardingViewController: UIViewController {
         return min(max(page, 0), onboardingPageControl.numberOfPages)
     }
 
+    private func setupBranding() {
+        eventNameLabel.text = EventBranding.current.name
+        
+        onboardingPageControl.currentPageIndicatorTintColor = EventBranding.current.color
+        
+        continueButton.setTitleColor(EventBranding.current.color, for: .normal)
```

Comment:
it would be cool to pair these 2 calls into one method with 1 color arg, since they depend on each other